ardere/step_functions.py

```diff
@@ -47,6 +56,17 @@ class StepValidator(Schema):
     env = fields.Dict()
     docker_series = fields.String(missing="default")
 
+    name_validator = re.compile("([:\*]+)")
```

Comment:
The name led me to believe this validated names are correct, but its the opposite. Maybe 'invalid_name'? Also, it probably shouldn't be on the class like this as the Schema class does special magic on setup for these names and is expecting them to be fields in the JSON, not additional helper vars.